schemas/com.snowplowanalytics.snowplow.media/ad_click_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event fired when the user clicked on the ad.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "ad_click_event",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "percentProgress": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "The percent of the way through the ad",
```

Comment:
Just checking - we round this in the tracker implementations right?

Reply:
Good point! We actually didn't before, I just added the rounding.